testing/test_dhcp.out

```diff
@@ -5,5 +5,5 @@ DHCP Tests
 02: ['02:ipaddr:TimeoutError']
 Device 1 ip triggers: 1 0
 Device 2 ip triggers: 0 0
-Device 3 ip triggers: 0 1
+Device 3 ip triggers: 1 1
```

Comment:
I feel like we should have a more explicit test for this... this seems very indirect/hidden.  The ip_triggers file has a list of the ip addresses used, so seems like we should be able to awk|uniq that file to make sure there's ==2 different addresses?